system-contracts/contracts/EvmInterpreterFunctions.template.yul

```diff
@@ -1448,7 +1402,7 @@ function performCreate2(evmGas, oldSp, isStatic) -> evmGasLeft, sp, result, addr
 
     {
         let hashedBytecode := keccak256(add(MEM_OFFSET_INNER(), offset), size)
-        mstore8(0, 0xFF)
+        mstore(0, 0xFF000000000000000000000000000000000000000000000000000000000000)
```

Comment:
Two 0s are missing here.
It has 31 bytes instead of 32.

Reply:
Nice catch. I have fixed this.